ui/src/app/pages/login/account-creation/account-creation-tos.tsx

```diff
@@ -0,0 +1,195 @@
+                    onRenderSuccess={index === numPages - 1 ? () => this.handleLastPageRender() :
+                      undefined}
+                  />
+                ),
+              )
+            }
+          </Document>
+        </div>
+        <FlexRow
+          style={{display: 'inline-flex', padding: '1rem', maxWidth: '1000px', margin: 'auto'}}>
+          <div style={{flex: 3}}>
+            <div style={{...styles.noticeText, marginBottom: '.5rem', height: '3rem'}}>
+              By clicking here and moving to the Registration step, you acknowledge that you
+              understand the terms of this agreement and agree to abide by them.
+              {!hasReadEntireTos && <span><br/>
+              <a data-test-id='scroll-to-bottom'
+                 onClick={() => this.scrollToBottom()}
+              >Scroll to bottom</a></span>}
+            </div>
+            <div style={{marginBottom: '.25rem'}}>
+              <CheckBox data-test-id='privacy-statement-check'
+                        checked={false}
+                        disabled={!hasReadEntireTos}
+                        onChange={checked => this.setState({hasAckedPrivacyStatement: true})}
+                        style={styles.checkbox}
+                        labelStyle={hasReadEntireTos ?
+                          styles.checkboxLabel :
+                          styles.disabledCheckboxLabel}
+                        wrapperStyle={{marginBottom: '0.5rem'}}
+                        label={<span>
+                I have read and understand the <i>All of Us</i> Research Program Privacy Statement.</span>}
+              /></div>
+            <div>
+              <CheckBox data-test-id='terms-of-service-check'
+                        checked={false}
+                        disabled={!hasReadEntireTos}
+                        onChange={checked => this.setState({hasAckedTermsOfService: true})}
+                        style={styles.checkbox}
+                        labelStyle={hasReadEntireTos ?
+                          styles.checkboxLabel :
+                          styles.disabledCheckboxLabel}
+                        wrapperStyle={{marginBottom: '0.5rem'}}
+                        label={<span>
+                I have read and understand the <i>All of Us</i> Research Program Terms of Use described above.</span>}
+              /></div>
+          </div>
+          <FlexColumn style={{paddingLeft: '3rem', alignItems: 'center', justifyContent: 'center'}}>
+            <Button data-test-id='next-button'
+                    style={{width: '5rem', height: '2rem', margin: '.25rem .5rem .25rem 0'}}
+                    disabled={!hasReadEntireTos || !hasAckedPrivacyStatement ||
+                    !hasAckedTermsOfService}
+                    onClick={() => this.props.onComplete()}>
+              Next
+            </Button>
+          </FlexColumn>
+        </FlexRow>
+      </FlexColumn>;
+    }
+  });
+
+export default AccountCreationTos;
```

Comment:
Should the TOS component itself live somewhere outside of the account creation flow? I thought we were planning to eventually have some kind of client-side interceptor to redirect users who hadn't submitted the TOS to do so (i.e. for handling future potential TOS updates). If that work is not planned for now, probably fine to leave as-is.